src/components/modal/alarmModal.tsx

```
@@ -0,0 +1,49 @@
+import { useEffect } from 'react';
+import { useInView } from 'react-intersection-observer';
+import ClipLoader from 'react-spinners/ClipLoader';
+
+import { useGetAlarm } from '@/hooks/alarm/useGetAlarm';
+
+import Alarm from '../alarmModal/alarm';
+import Modal from '../common/modal';
+
+import Error from '@/pages/common/Error';
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)